test/references/CKAN Gold Prices.txt

```diff
@@ -1,12 +1,12 @@
 register(DataDep(
     "Gold Prices in London 1950-2008 (Monthly)",
     """
-    
-Website: https://demo.ckan.org/dataset/gold-prices
-Author: 	Bundesbank
-Maintainer: 	Rufus Pollock
-License: 	Open Data Commons Public Domain Dedication and License (PDDL)
-Date of Creation: 	July 18, 2017
+Website:	https://demo.ckan.org/dataset/gold-prices
+Author:	Bundesbank
```

Comment:
I just notices.
Why don't these ones get tabs?
Tabs or no tabs I don't mind but consistency is good